Index.html

```diff
@@ -77,5 +77,170 @@ <h1>I'm Kerlos Baha.<br />Glad to see <br />you!</h1>
       </section>
       <span class="shape"> </span>
     </main>
+    <div class="space"></div>
+
+    <!-- first work page -->
+
+    <section class="repeated-section">
+      <img class="repeated-img" src="/picture/Snapshoot Portfolio.png" alt="" />
+      <p class="repeated-p1">Tonic</p>
+      <ul class="repeated-ul">
+        <li class="repeated-li1">CANOPY</li>
+        <li class="repeated-li2">Back End Dev</li>
+        <li class="repeated-li3">2015</li>
+      </ul>
+      <p class="repeated-p2">
+        A daily selection of privately personalized reads; no accounts or
+        sign-ups required.
+      </p>
+      <ul class="repeated-ul2">
+        <li class="repeated-li4">HTML</li>
+        <li class="repeated-li5">CSS</li>
+        <li class="repeated-li6">JavaScript</li>
+      </ul>
+      <button class="see-button">See project</button>
+    </section>
+
+    <div class="space-2"></div>
+    <!-- second work page -->
+    <section class="repeated-section-2">
+      <img
+        class="repeated-img"
+        src="/picture/Snapshoot Portfolio2.png"
+        alt=""
+      />
```

Comment:
Currently, none of your images are displaying in the browser:

<img width="186" alt="image" src="https://user-images.githubusercontent.com/37116322/227353412-f99d398d-5cc6-456a-926a-64319a640ae3.png">

Please check the file paths to all images in your project and ensure that you use a relative file path by adding a `.`  before the path to the file E.G `./path-to-file`🙏.

Please also update your image file names, so none of them contain spaces and brackets(). This is what is causing your images not to load👍